iosApp/iosApp/Explore/RepoListView.swift

```diff
@@ -7,23 +7,68 @@
 //
 
 import SwiftUI
+import ExploreShared
 
 struct RepoListView: View {
     @StateObject var viewModel = RepoListViewModel()
 
     var body: some View {
-        ForEach(self.viewModel.repoList, id: \.name) { repo in
-            VStack {
-                Text(repo.name)
-            }.padding(.bottom, 10)
-        }.onAppear {
-            Task {
-                await self.viewModel.loadRepos()
+        let list = self.viewModel.repoList
```

Comment:
It's not common practice to define variables inside the `body`. We can define a computed property under the viewModel `StateObject` that returns the `viewModel.repoList` || `viewModel.isLastPage`. Example: `var list: [Repo] {
        self.viewModel.list
    }`